texts/en_US.properties

```diff
@@ -2,15 +2,15 @@ geyser.auth.already_loggedin={0} is already logged in\!
 geyser.auth.floodgate.bad_key=Error while reading Floodgate key file
 geyser.auth.floodgate.encrypt_fail=Failed to encrypt message
 geyser.auth.floodgate.loaded_key=Loaded Floodgate key\!
-geyser.auth.login.floodgate=Attempting to login using floodgate mode... authentication will be encrypted.
-geyser.auth.login.form.details.desc=Enter the credentials for your Minecraft\: Java Edition account below.
+geyser.auth.login.floodgate=Attempting to login using floodgate mode. Authentication will be encrypted.
+geyser.auth.login.form.details.desc=Enter the credentials for your Minecraft\: Java Edition account in the fields below.
 geyser.auth.login.form.details.email=Email/Username
 geyser.auth.login.form.details.pass=Password
 geyser.auth.login.form.details.title=Login Details
 geyser.auth.login.form.disconnect=Login is required
 geyser.auth.login.form.notice.btn_disconnect=Disconnect
 geyser.auth.login.form.notice.btn_login=Login with Minecraft
-geyser.auth.login.form.notice.desc=You need a Java Edition account to play on this server.
+geyser.auth.login.form.notice.desc=You will need a Minecraft: Java Edition account to play on this server.
```

Comment:
The context this string is presented makes more sense in its current form (when you are disconnected when not providing credentials)